### PR TITLE
lab: Add lab 4 - for-loops

### PR DESCRIPTION
**Pull request by juliaogris**
Add lab 4, the last "fundamental" labs about for-loops, with 8 challenges

- 1/8 Welcome & intention: let's build this line animation
- 2/8 Intro to for loops: re-write as while loop
- 3/8 Zebra crossing drawing & animation with for loops
- 4/8 Fun/free circle drawing with samples similar to ifs/rectangles
- 5/8 Text based for loop from scratch: 4 simple sequences
- 6/8 Bubble circle rising with jitter to surface: for-loop animation &
     randomness from scratch
- 7/8 Line animation - "let's think about it"
- 8/8 Line animation - step-by-step animation

Additional updates:

- Refactor: Move lines sample from loops lab to for-loops lab
- CSS: Tweak space between code-block and paragraph in hints
- Fix the markdown in the loops lab welcome page which had some left-over
HTML copy-pasta.
- make: Remove accidentally duplicated .PHONY target

---

Preview:
https://lab.evytest.dev/#welcome-forloops

### CODE REVIEW

File: frontend/lab/samples/forloops/forcircles.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+```evy
+for x := range 1000
+	// ...
+  	// eyes
+  	offset := x % 12 / 8
+    move 31+offset 70
+    circle 2
+    move 49+offset 70
+	// ...
```
Comment (camh-) [inline frontend/lab/samples/forloops/forcircles.md:54]: some funny indenting in this block? tabs vs spaces?

File: frontend/lab/samples/forloops/forsequence.evy [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (camh-) [inline frontend/lab/samples/forloops/forsequence.evy:1]: Is this intentionally an empty file? Not completely sure I understand it's purpose so maybe it's just needed as a placeholder :shrug:

Comment (juliaogris) [reply frontend/lab/samples/forloops/forsequence.evy:1]: yeah, it is needed so I don't get a 404 on frontend side. I should just add a little extra config attribute in sample.json... noted. next time.

File: frontend/lab/samples/forloops/forcircles.md (hunk 1 above)
Comment (juliaogris) [reply frontend/lab/samples/forloops/forcircles.md]: ah, yes. i should run the formatter over all these too I suppose.
ah no. that's not going to work. that's why i haven't been doing it:

```
 line 5 column 9: illegal character "❓"
```

would be good to have a  "loose" evy fmt or so....
i also purposefully only used 2-space indentation for the tight columns in the labs.

anyway, fixed manually for now.

**Review by camh- — APPROVED** [review]
:green_book:

**Pull request merged**